e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_1_spec.js

```diff
@@ -441,6 +441,38 @@ describe('Keyboard Shortcuts', () => {
         cy.get('#moreDmModal').should('not.exist');
     });
 
+    it('MM-T4452 - CTRL/CMD+SHIFT+. Expand or collapse RHS', () => {
+        // # Post a message and open RHS
+        const message = `hello${Date.now()}`;
+        cy.postMessage(message);
```

Comment:
```suggestion
        cy.postMessage(MESSAGES.TINY);
```
we have a fixture for this can you use that one `fixtures/messages`